src/docs/components/quote/QuoteDoc.js

```diff
@@ -17,7 +21,7 @@ export default class QuoteDoc extends Component {
         }>
 
         <section>
-          <p>A quote with a colored border.</p>
+          <p>{DESC} A quote with a colored border.</p>
```

Comment:
This PR adds a description for the Quote component as there was not currently a description in the docs. https://github.com/karatechops/grommet-docs/blob/256c6d3452a346086592b79f5503a1a9604e6247/src/docs/components/quote/QuoteDoc.js#L10-L12